### PR TITLE
Misc Fixes

### PR DESCRIPTION
**Pull request by marcbonnici**

### CODE REVIEW

File: devlib/host.py [hunk 1]
```diff
@@ -55,7 +56,10 @@ def pull(self, source, dest, timeout=None, as_root=False): # pylint: disable=unu
             for each_source in iglob(source):
                 shutil.copy(each_source, dest)
         else:
-            shutil.copy(source, dest)
+            if os.path.isdir(source):
+                copy_tree(source, dest)
```
Comment (setrofim) [inline devlib/host.py]: Why not use `shutil.copytree` here?

File: devlib/__init__.py [hunk 2]
```diff
@@ -56,7 +56,7 @@
 from devlib.utils.version import get_commit as __get_commit
 
 
-__version__ = '1.0.0'
+__version__ = '1.0.dev1'
```
Comment (setrofim) [inline devlib/__init__.py]: Please expand commit message to state the reason for this change.

Comment (marcbonnici) [reply devlib/__init__.py]: Updated.

File: devlib/host.py (hunk 1 above)
Comment (marcbonnici) [reply devlib/host.py]: `shutil.copytree` does not support copying into an existing directory structure whereas the implementation from `distutils` does, therefore to try and mimic the behaviour of other targets as closely as possible I opted for this implementation.

Comment (setrofim) [reply devlib/host.py]: OK that makes sense. It's worth leaving a comment to that effect in the code.

Comment (marcbonnici) [reply devlib/host.py]: Updated.

**Pull request merged**